lol-apps/Crypto/Lol/Applications/Benchmarks/KHPRFBenches.hs

```diff
@@ -19,26 +19,26 @@ Benchmarks for KeyHomomorphicPRF.
 {-# LANGUAGE ScopedTypeVariables   #-}
 {-# LANGUAGE TypeFamilies          #-}
 
-module Crypto.Lol.Applications.Benchmarks.KHPRFBenches
-( khprfBenches, main ) where
+module Crypto.Lol.Applications.Benchmarks.KHPRFBenches (khprfBenches) where
+
+import Control.Applicative
+import Control.DeepSeq
+import Control.Monad.Random hiding (fromList, split)
 
 import Crypto.Lol hiding (replicate)
 import Crypto.Lol.Types
 import Crypto.Lol.Applications.KeyHomomorphicPRF
-import Crypto.Lol.Cyclotomic.Tensor.CPP
-
-import Control.Applicative
-import Control.Monad.Random hiding (fromList, split)
+import Crypto.Lol.Reflects
 
 import Criterion.Main
 
 type M = F64
 type N = 1
 type Q = 256
 type P = 2
-type Rq = Cyc CT M (ZqBasic Q Int64)
-type Rp = Cyc CT M (ZqBasic P Int64)
-type Gad = BaseBGad 2
```

Comment:
I just noticed that `Gad` is now out of the picture, and is treated as a parameter (by proxy) in `khprfBenches`. This is to make it look more like `[default]sheBenches`? I guess there's no harm in that, but maybe it's better if `defaultKHPRFBenches` uses `BaseBGad 2` instead of being provided a gadget proxy (it's supposed to be a sensible default, after all). I'm open to arguments against this too.

Reply:
Yes, the sole purpose was to make it look similar to defaultSHEBenches.  It was a relatively simple change to the code and I figured it allows for more gadget options in `lol-cpp/benchmarks/BenchAppsCPPMain.hs`.